### PR TITLE
uploadrocket.net

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [X] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/139027

### Add your comment and screenshots

 - https://github.com/AdguardTeam/AdguardFilters/commit/5db1d28b9478c44e33fee3313feb8d1521d89410

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: FrenchFilter/sections/allowlist.txt [hunk 1]
```diff
@@ -82,7 +82,7 @@ $script,third-party,domain=europixhd.net,badfilter
...
...|uploadrar.com|uploadrocket.net|upornia.com|uppit.com|uppom.live|upstream.to|uptobox.com|uptostream.com|upvid.co|upvid.host|upvid.live|upvid.net|upvid.pro|uqload.com|uqload.org|uqvid.org|urlbeat.net|userload.co|userscloud.com|vadbam.com|vanfem.com|vcrypt.net|vedbam.xyz|vedbom.com|vev.io|vev.red|vid.ag|vidbam.org|vidbem.com|vidbm.com|vidbom.com|vidcloud.co|vidembed.cc|vidembed.io|vidembed.me|video-embed-imdb.xyz|videobin.co|videospider.in|videovard.sx|videowood.tv|vidhd.fun|vidhd.net|vidhd.site|vidlo.us|vidlooz.com|vidnode.net|vido.fun|vidoza.co|vidoza.net|vidplay.ch|vidshar.org|vidshar.tv|vidshare.tv|vidspeed.net|vidspeeds.com|vidsrc.me|vidstream.kim|vidstream.to|vidstreaming.io|vidtodo.com|vidup.io|vidup.tv|viewsb.com|vimple.me|vipbox.lc|vipbox.live|vipleague.cc|vipleague.im|vipleague.ws|voe.sx|vudeo.io|vudeo.net|vup.to|w4files.ws|waaw.tv|waaw1.tv|watchcartoonsonline.eu|watchvideo.us|whackyvidz.com|wstream.to|wstream.video|xfilesharing.com|xrares.com|xstreamcdn.com|xvideosharing.com|xxjave.xyz|yggtorrent.site|yodbox.com|youdbox.com|youdbox.net|younetu.com|yourbittorrent.com|yourupload.com|youtubedownloader.cx|youwatch.org|yts.mx|yts.pm|yyv.co|zipansion.com|zippyshare.com,badfilter
```
Comment (zloyden) [inline FrenchFilter/sections/allowlist.txt:85]: This rule is disabled by `$badfilter`, so it should be the same as the original one. Don't remove the domain.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: FrenchFilter/sections/allowlist.txt (hunk 1 above)
Comment (Alex-302) [reply FrenchFilter/sections/allowlist.txt:85]: @zloyden better check if it works. `$badfilter` should not be sensitive to domains order.

Comment (krystian3w) [reply FrenchFilter/sections/allowlist.txt:85]: Also smed79 deleted/added some domains:

```diff
@@ deletion: @@
-- fflares.com
-- anavids.com
-- kvid.org
-- k-streaming.cc
-- k-streaming.com
-- moviefishers.org
-- mp3im.org
-- newdmn.icu
-- sonyoutube.com
-- streamsforu.com
-- telerium.tv
-- topeuropix.com
-- tvrush.eu
-- videovard.sx
@@ addition: @@
++ 2embed.to
++ 745mingiestblissfully.com
++ 9goaltv.in
++ al.ly
++ asianplay.pro
++ dotvideo.cc
++ earthquakecensus.com
++ justameanlessdomain.com
++ novaflix.io
++ nstream.to
++ povwideo.cc
++ score911.cc
++ stapadblockuser.click
++ stapadblockuser.xyz
++ thevideo.me
++ uv9ieb2ohr.com
++ vcstream.to
++ vido.cool
++ vido.cx
++ vido.lol
++ vipleague.me
```

Comment (zloyden) [reply FrenchFilter/sections/allowlist.txt:85]: https://github.com/AdguardTeam/AdguardFilters/pull/141689#discussion_r1093468617 looks like it works as expected even  if we remove the domain.

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@piquark6046 please update the rule

**Comment by piquark6046** [comment]
Updated. @Alex-302

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**